src/ucp/api/ucp.h

```diff
@@ -1110,6 +1110,29 @@ struct ucp_tag_recv_info {
  * The structure @ref ucp_request_param_t is used to specify datatype of
  * operation, provide user request in case if external request should be used,
  * set completion callback and custom user data passed to this callback.
+ * 
+ * Example:
+ * 
+ * ucs_status_ptr_t send_data(ucp_ep_h ep, void *buffer, size_t length,
+ *                        ucp_tag_t tag, void *request)
```

Comment:
- indentation
- can you pls use http://www.doxygen.nl/manual/commands.html#cmdcode

Reply:
also pls comment what this example is doing

Reply:
done, example is updated